diff --git a/src/go2_perception/launch/go2_pointcloud.launch.py b/src/go2_perception/launch/go2_pointcloud.launch.py
index 4c61a34..f20f720 100644
--- a/src/go2_perception/launch/go2_pointcloud.launch.py
+++ b/src/go2_perception/launch/go2_pointcloud.launch.py
@@ -18,8 +18,8 @@ def generate_launch_description():
                 ('scan', '/scan')
                 ],
             parameters=[{
-                'target_frame': 'base_footprint',
-                'transform_tolerance': 0.05,
+                'target_frame': 'base_link',
+                'transform_tolerance': 0.3,
                 # 高度按需调整
                 'min_height': 0.1,
                 'max_height': 0.5,
diff --git a/src/go2_perception/src/pointcloud_to_laserscan_node.cpp b/src/go2_perception/src/pointcloud_to_laserscan_node.cpp
index f77f325..ec2bfc9 100644
--- a/src/go2_perception/src/pointcloud_to_laserscan_node.cpp
+++ b/src/go2_perception/src/pointcloud_to_laserscan_node.cpp
@@ -49,7 +49,7 @@
 #include <utility>
 
 #include "sensor_msgs/point_cloud2_iterator.hpp"
-#include "tf2_sensor_msgs/tf2_sensor_msgs.hpp"
+#include "tf2_sensor_msgs/tf2_sensor_msgs.h"
 #include "tf2_ros/create_timer_ros.h"
 #include "tf2/LinearMath/Transform.h"
 #include "geometry_msgs/msg/transform_stamped.hpp"
diff --git a/src/go2_slam/config/mapper_params_online_async.yaml b/src/go2_slam/config/mapper_params_online_async.yaml
index 5338da4..0dac582 100644
--- a/src/go2_slam/config/mapper_params_online_async.yaml
+++ b/src/go2_slam/config/mapper_params_online_async.yaml
@@ -11,7 +11,7 @@ slam_toolbox:
     # ROS Parameters
     odom_frame: odom
     map_frame: map
-    base_frame: base_footprint 
+    base_frame: base_link 
     scan_topic: /scan
     mode: mapping #localization
 
